### PR TITLE
Elena/make Map and Tile components

### PR DESCRIPTION
**Pull request by echurilov**
Started out making a really basic table of map tiles using a nice ocean texture I found.

### CODE REVIEW

**Comment by echurilov** [comment]
Todo:
- [ ] Pass background image as prop on `<Tile>`
- [ ] Get which backgrounds to use from (dummy) map state?

File: src/Map.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Tile from './Tile';
+import './Map.css';
+
+function Map() {
+    return (
+      <table className="Map">
+        <tbody>
+          <tr><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /><Tile /></tr>
```
Comment (beshaya) [inline src/Map.tsx]: Surely we can use a for loop here :P

**Review by beshaya — COMMENTED** [review]
Let's chat about how to lay out the grid - Shannon wants the map to be filled in with colors by default, but textures have to be logged by players.

File: src/Map.tsx (hunk 1 above)
Comment (echurilov) [reply src/Map.tsx]: Haha, my embarrassing interim state exposed! Yes, we can :P

Comment (echurilov) [reply src/Map.tsx]: I think my idea for where to go from here may actually have been to map over an array of tiles...? What are your thoughts for how we're going to store map state?

**Comment by echurilov** [comment]
Mucked around trying to familiarize myself with typescript. Also added that for loop :P
This is probably good for now and I'll do stuff like "tiles can have different things in them" in a different PR

**Pull request merged**